packages/@local/tsconfig/dom.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Adds flexibility: we can inherit non-React tsconfig with DOM

Reply:
Since we have this, we could use it in `@blockprotocol/*`, since none of them contain JSX

Reply:
Good idea! https://github.com/blockprotocol/blockprotocol/pull/585/commits/36f6bba8d3ae285699e668a5cada77990cf5ed79